lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -9,7 +9,7 @@ namespace :update do
         puts "== #{engine.name} =="
         system("which yarn >/dev/null") || abort("\n== You have to install yarn ==")
         system("yarn set version 1.22.18") || abort("\n== yarn failed to set version to 1.22.18 in #{engine.path} ==") if RUBY_PLATFORM.include?("s390x")
-        system("yarn") || abort("\n== yarn failed in #{engine.path} ==")
+        system("yarn --frozen-lockfile") || abort("\n== yarn failed in #{engine.path} ==") # Switch --frozen-lockfile to --immutable once s390x is off of yarn 1.
```

Comment:
This code also runs locally during `rake update:ui`.  In those cases we also want to use the frozen lockfile, but during development if someone is working on trying to update the lockfile, I think this might get in the way?  Then again, maybe not, because I think developers will modify the lockfile by manually doing a yarn install, then committing that.

@GilbertCherrie can you comment here if that is a fair statement and if you think there are any issues with committing this?

Reply:
This should be good.